walk-in-interview/src/main/webapp/account/applicant/edit/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../../../style.css">
+  </head>
+
+  <body>
+    <div class="header-container">
+      <button id="back"></button>
+      <input id="submit">
+    </div>
+
+    <div class="content-container">
+      <ul>
+        <li>
+          <label id="name-label" for="name"></label>
+          <input id="name" name="name">
+        </li>
+
+        <li>
+          <label for="skills-title"></label>
+          <div id="skills-title" name="skills-title"></div>
+          <ul id="skills"></ul>
+        </li>
+      </ul>
+    </div>
+
+    <div class="error-message" id="error-message"></div>
```

Comment:
Is it intentional that this error locates between skill title and skill list?

Reply:
the below skill list is just a hidden template which will be utilized in the `<ul id="skills"></ul> above